public/docs/_examples/template-syntax/ts/app/app.component.html

```diff
@@ -351,21 +351,22 @@
 <hr><h2 id="two-way">Two-way Binding</h2>
 <div id="two-way-1">
   <!-- #docregion two-way-1 -->
-  <my-sizer [(size)]="fontSize"></my-sizer>
-  <div [style.font-size.px]="fontSize">Resizable Text</div>
+  <my-sizer [(size)]="fontSizePx"></my-sizer>
```

Comment:
Actually, this is mainly to avoid a conflict with a string property named `fontSize` whose initial value is `"large"`.

It seemed to me that since the property is being assigned to `[style.font-size.px]` (see the old line 355 above) it could naturally be named `fontSizePx` (which is the camel-case equivalent of the style attribute it is being assigned to).